docs/examples/network_profile/README.md

```diff
@@ -0,0 +1,75 @@
+# Network profile examples
+
+To run this examples you need to [create a Tenant](../README.md#tenant) and [set the environment variable](../README.md#pipelinerun) `TENANT_NAMESPACE` accordingly.
+
+## Valid profile configuration
+
+To run these examples apply `profiles.yaml` to your Steward system namespace, e.g. `steward-system`:
+
+```
+kubectl -n steward-system apply -f profiles.yaml
+```
+
+### Network profile with internet access
+
+This example pipeline run uses a network profile which allows internet access.
+
+```
+kubectl -n  $TENANT_NAMESPACE apply -f pipelinerun_network_internet.yaml
+```
+
+The result is a successful run.
+```
+NAME                             STARTED   FINISHED   STATUS     RESULT          MESSAGE
+network-profile-internet-qgqvf   **        **         finished   success         Pipeline completed with result: SUCCESS
+```
+
+### Network profile with no network access
+
+This example pipeline run uses a network profile which allows no network access.
+
+```
+kubectl -n  $TENANT_NAMESPACE apply -f pipelinerun_network_blocked.yaml
+```
+
+The result is a run failing with a content error.
+
+```
+NAME                             STARTED   FINISHED   STATUS     RESULT          MESSAGE
+network-profile-blocked-qgjfp    **        **         finished   error_content   Command ['git' 'clone' 'https://github.com/SAP-samples/stewardci-example-pipelines' '.'] failed w...
+```
+
+### Network profile with unknown profile
+
+This example pipeline run tries to use a nonexistent network profile.
+
+
+```
+kubectl -n  $TENANT_NAMESPACE apply -f pipelinerun_network_unknown.yaml
+```
+
+The pipeline run result is `error_config`, which indicates that the pipeline run specification was erroneous.
+
+```
+NAME                             STARTED   FINISHED   STATUS     RESULT          MESSAGE
+network-profile-unknown-9p8v8    **        **         finished   error_config    ERROR: preparing failed ...
+```
+
+## Invalid profile configuration
+
+If the definition of the network profile config map is inconsistent (e.g. not existing default profile), all pipeline runs will fail with result `error_infra`.

```

Comment:
We should mention the effect of feature flag `RetryOnInvalidPipelineRunsConfig`.

Reply:
Done